nilrt_snac/_Configs/_NtpConfig.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import subprocess
+
+from nilrt_snac._Configs._BaseConfig import _BaseConfig
+from nilrt_snac._Configs._ConfigFile import _ConfigFile
+
+from nilrt_snac import logger
+from nilrt_snac.OpkgHelper import opkg_helper
+
+
+class _NtpConfig(_BaseConfig):
+    def __init__(self):
+        self._config_file = _ConfigFile("/etc/ntp.conf")
+        self._opkg_helper = opkg_helper
+
+    def configure(self, args: argparse.Namespace) -> None:
+        logger.info("Configuring NTP...")
+        dry_run: bool = args.dry_run
+        self._opkg_helper.install("ntp")
+
+        logger.debug("Switching ntp servers to US mil.")
+        if self._config_file.contains("natinst.pool.ntp.org"):
+            self._config_file.update("^.*natinst.pool.ntp.org.*$", "")
+
+        if not self._config_file.contains("server 0.us.pool.ntp.mil iburst maxpoll 16"):
+            self._config_file.add("server 0.us.pool.ntp.mil iburst maxpoll 16")
+
```

Comment:
I was thinking that the dry-run's purpose was to aid devloper testing on non-NILRT systems (hence why it skips the prereq steps). But these `configure()` methods seem to keep modifying the system state in breaking ways. What is the purpose of dry-run then?

Reply:
ConfigFile.save() takes a dry_run parameter. Until save is called, the changes are only in memory and are only written to disk if dry_run is False when save is called.

Reply:
OK. I guess that makes sense. I think I saw other places in the Config files where package installation or removal is performed regardless of dry_run. I understand that mimicing that behavior might mean that you cannot dry run subsequent steps that use those package contents; but I think it is more important that the dry-run not make changes to your system state.